pkg/devfile/parser/reader_test.go

```diff
@@ -72,7 +72,9 @@ func TestReadAndParseKubernetesYaml(t *testing.T) {
 		name                string
 		src                 YamlSrc
 		fs                  *afero.Afero
+		testParseYamlOnly   bool
```

Comment:
where are we setting this? We are using it but i dont see it being set a value

Reply:
https://github.com/devfile/library/blob/88946b65ac8d82e7e886b825ff76d8d1bc51893a/pkg/devfile/parser/reader_test.go#L177